### ISSUE TITLE
Running/Loading Slowly and inconsistent numbers tracking

### ISSUE THREAD

**Issue by nyac1998** [blackjackkent/RPThreadTrackerV3.FrontEnd#182]
## Expected Behavior

correct load times 

numbers displayed for what is owed/how much owed to me/queued

## Current Behavior

Taking more than 10 minutes to load the page.

i can't really tell sometimes if the numbers are accurate. i'll reload and get three different sets of numbers for everything

## Possible Solution

Just a question of if doing the patreon would fix the slow loading because of the terabyte usage or curious if there's just a bug happening right now. 

i don't know how that would get fixed honestly, i just reload it several times until the numbers stay consistent

## Steps to Reproduce (for bugs)

1. reload the page
2. saving a change in the page
3. marking threads queued/deleted anything

1. initial reload of the page
2. second reload to see if the numbers change or not

## Context

this is the only thing i use to do replies, and when it doesn't work i'm kind of stuck not being able to do anything. which sucks because i do really love using this site. 

this is less of an impact i just figured i'd document that it's happening

## Your Environment

* Browser name and version (e.g. Chrome 39, node.js 5.4):
* Operating System and version (desktop or mobile):
using desktop on chrome, Version 121.0.6167.161.

**Comment by blackjackkent** [comment]
Hey - thanks for getting in touch about this! I'm happy to take a look into it and see what's going on. A couple extra questions:

* What is your username on the tracker?
* When did this issue start happening?

Thanks!

**Comment by nyac1998** [comment]
Username is: wvsteria
the slow loading just started this week. the numbers thing has been since i
started using it months ago

**Comment by blackjackkent** [comment]
OK, looking at your account, I am guessing the slow loading results from the fact that you have 364 threads tracked and none of them archived, so the tracker is trying to make 364 different calls to the Tumblr API to fetch all of the information. (This is sadly a limitation of Tumblr's API that I have to request information about each thread individually.) You'll probably see a performance improvement if you archive threads you are not interested in tracking anymore so they can be fetched separately (and more slowly) on the archive view and keep your active view more snappy. 

I can't duplicate the inconsistent numbering on my end but my guess would be that it's related to the same cause. 

If reducing your number of active threads doesn't resolve one of these issues, can you give me a ping at rosalind@blackjack-software.com with, if you're willing, your login information for the app? (You can change your password to give it to me and then change it back afterwards if you want.) I can log into your account and see if I can figure out any other potential issues. But my guess is that you need to archive the threads that aren't active anymore for better performance.

**Comment by nyac1998** [comment]
So I am actively using almost all the threads. Would I have to be a Patreon
or something to get more terabytes to make it run smoother? I was thinking
that could be the issue tbh just wasn't sure if using the patreon would
actually give it more space or something to help it.

**Comment by blackjackkent** [comment]
Ahh. No, I'm afraid that Patreon contributions don't unlock a higher tier, they just help provide me with the resources to host the site in its current state. :(

**Comment by nyac1998** [comment]
ohh :( okay well thanks anyway! I appreciate you clarifying it for me.

**Comment by blackjackkent** [comment]
Of course. Thank you for getting in touch. I will of course continue to work on performance improvements where I can. :)

**Comment by nyac1998** [comment]
The site is great! And I truly appreciate you making something like this! I
hope you have a great day! 😁
